src/components/BottomSheet/BottomSheetLabel.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import colors from "../../theme/colors";
+
+import { Text } from "../Text";
+import { StatusBadge } from "../StatusBadge";
+
+const LabelText = styled(Text)`
+  text-transform: uppercase;
+`;
+
+const BottomSheetLabel = ({ variant, children }) => {
```

Comment:
Can you please spread additional props that are passed to `BottomSheetLabel`? I would like if you could please pass a `className` here: https://github.com/ticketmaster/aurora/pull/318/files#diff-6ca6b58c9af788225a35455dbf68d85aR116